sdk/src/main/java/com/akkaserverless/javasdk/lowlevel/ValueEntityHandlerFactory.java

```diff
@@ -25,7 +25,7 @@
  * com.akkaserverless.javasdk.valueentity.ValueEntity @ValueEntity} and similar annotations should
  * be used.
  */
-public interface ValueEntityFactory {
+public interface ValueEntityHandlerFactory {
```

Comment:
This is rather an ValueEntityHandlerFactory. Moreover, I needed to claim that old name. :-)